### PR TITLE
Add slic3r - Prusa edition version 1.36

### PR DESCRIPTION
**Pull request by schwabe**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [ x] `brew cask audit --download {{cask_file}}` is error-free.
- [ x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [ x] The commit message includes the cask’s name and version.

Additionally, if **updating a cask**:

- [ ] [If the `sha256` changed but the `version` didn’t][version-checksum],  
      provide public confirmation by the developer: {{link}}

Additionally, if **adding a new cask**:

- [x ] Named the cask according to the [token reference].
- [ x] `brew cask install {{cask_file}}` worked successfully.
- [ x] `brew cask uninstall {{cask_file}}` worked successfully.
- [ x] Checked there are no [open pull requests] for the same cask.
- [x ] Checked the cask was not already refused in [closed issues].
- [ x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/slic3r-prusa.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cask 'slic3r-prusa' do
```
Comment (commitay) [inline Casks/slic3r-prusa.rb]: This should be `prusa-slic3r`.

File: Casks/slic3r.rb [hunk 2]
```diff
@@ -8,5 +8,7 @@
   name 'Slic3r'
   homepage 'http://slic3r.org/'
 
+  conflicts_with cask: 'slic3r-prusa'
```
Comment (commitay) [inline Casks/slic3r.rb]: `prusa-slic3r`

**Review by commitay — CHANGES_REQUESTED** [review]

File: Casks/slic3r-prusa.rb (hunk 1 above)
Comment (schwabe) [inline Casks/slic3r-prusa.rb]: Okay, I probably missed something on the guid how to name casks, thanks!

**Review by commitay — APPROVED** [review]

**Comment by schwabe** [comment]
Damn it, everything that could have gone wrong has gone wrong here, like prusa realising a new version just when I opened the pull request. Thanks for fixing that for me

**Pull request merged**